app/src/main/java/ch/epfl/skysync/screens/Chat.kt

```diff
@@ -1,32 +1,34 @@
 package ch.epfl.skysync.screens
 
-import androidx.compose.foundation.background
-import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
+import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
+import ch.epfl.skysync.navigation.BottomBar
 
 @Composable
-fun ChatScreen() {
-  Box(
-      modifier = Modifier.fillMaxSize().background(Color.Red),
-      contentAlignment = Alignment.Center) {
-        Text(
-            text = "Chat",
-            fontSize = MaterialTheme.typography.h3.fontSize,
-            fontWeight = FontWeight.Bold,
-            color = Color.White)
-      }
+fun ChatScreen(navController: NavHostController) {
+  Scaffold(modifier = Modifier.fillMaxSize(), bottomBar = { BottomBar(navController) }) { padding ->
+    androidx.compose.material3.Text(

```

Comment:
It would be better to import it at the beginning and replace the "import androidx.compose.material.Text" with "import androidx.compose.material3.Text"

Reply:
Yes, I fixed it it was due to the import of material instead of material 3